src/values.rs

```diff
@@ -596,11 +589,11 @@ where
 /// # use bevy::prelude::Color;
 /// # use bevy_particle_systems::values::{CurvePoint, Curve};
 /// let curve = Curve::new(vec![CurvePoint::new(Color::BLACK, 0.0), CurvePoint::new(Color::WHITE, 1.0)]);
-/// assert_eq!(curve.sample(0.5), Color::rgba(0.5, 0.5, 0.5, 1.0));
+/// assert_eq!(curve.sample(0.5), Color::linear_rgba(0.5, 0.5, 0.5, 1.0));
 ///
 /// let three_color_curve = Curve::new(vec![CurvePoint::new(Color::BLACK, 0.0), CurvePoint::new(Color::WHITE, 0.5), CurvePoint::new(Color::BLACK, 1.0)]);
-/// assert_eq!(three_color_curve.sample(0.5), Color::rgba(1.0, 1.0, 1.0, 1.0));
-/// assert_eq!(three_color_curve.sample(0.75), Color::rgba(0.5, 0.5, 0.5, 1.0));
+/// assert_eq!(three_color_curve.sample(0.5), Color::linear_rgba(1.0, 1.0, 1.0, 1.0));
+/// assert_eq!(three_color_curve.sample(0.75), Color::linear_rgba(0.5, 0.5, 0.5, 1.0));
```

Comment:
Same here, shouldn't `Color::srgba()` be used here?

Reply:
Same as above.